descriptors/ModuleDescriptor-template.json

```diff
@@ -2304,8 +2304,9 @@
       "description" : "Internal permission set for fetching item(s)",
       "subPermissions": [
         "inventory-storage.items.item.get",
-        "circulation-item-storage.items.item.get",
+        "circulation-item-storage.item.collection.get",
         "circulation-item-storage.items.collection.get",
+        "circulation-item-storage.items.item.get",
```

Comment:
Is the permission needed or not needed ? Or am i missing something?

Reply:
Permission is needed. And this permission is already added in circ-item